uportal-war/src/main/java/org/jasig/portal/rest/PeopleRESTController.java

```diff
@@ -102,4 +103,18 @@ public ModelAndView getPerson(@PathVariable String username,
         return mv;
     }
 
+    @RequestMapping(value = "/people/me.json", method = RequestMethod.GET)
```

Comment:
Two questions:

* I have seen the word `self` used in REST APIs so commonly that I suspect it's idiomatic; do you have an objection to changing to `/people/self.json`?
* Do we really need to add this endpoint to both controllers?  Would it be enough to add it to the `PeopleRESTControllerV43`?